protos/sources/data-node/api/v1/trading_data.proto

```diff
@@ -731,7 +731,7 @@ message AccountsSubscribeRequest {
   // Party identifier
```

Comment:
These are sensible changes but I think perhaps we should leave them alone as this API is about to be deprecated anyway and this will probably break e.g. the trading bots. It's make the V2 API the perfect one and then people can change as they migrate